packages/carbon-component-mapper/src/files/checkbox.js

```diff
@@ -7,21 +7,35 @@ import { Checkbox as CarbonCheckbox, FormGroup } from 'carbon-components-react';
 
 import WithDescription from '../common/with-description';
 import prepareProps from '../common/prepare-props';
+import HelperTextBlock from '../common/helper-text-block';
 
-const Wrapper = ({ label, description, children }) => (
-  <FormGroup legendText={description ? <WithDescription labelText={label} description={description} /> : label}>{children}</FormGroup>
+const Wrapper = ({ label, description, children, helperText, error, showError }) => (
+  <FormGroup legendText={description ? <WithDescription labelText={label} description={description} /> : label}>
+    {children}
+    <HelperTextBlock helperText={helperText} errorText={showError && error} />
+  </FormGroup>
 );
 
 Wrapper.propTypes = {
   label: PropTypes.node,
   children: PropTypes.node,
-  description: PropTypes.node
+  description: PropTypes.node,
+  helperText: PropTypes.node,
```

Comment:
Are these new props part of the component typings?

Reply:
Yeah, they are in the formgroup definition.